contracts/DHTT.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.4.18;
+
+import 'zeppelin-solidity/contracts/token/ERC20/MintableToken.sol';
+
+contract DHTT is MintableToken {
```

Comment:
CappedToken?

Reply:
We do not need `CappedToken` since we have `finishMinting` call in constructor.